vitrivr-engine-core/src/main/kotlin/org/vitrivr/engine/core/database/blackhole/descriptors/BlackholeDescriptionProvider.kt

```diff
@@ -0,0 +1,21 @@
+package org.vitrivr.engine.core.database.blackhole.descriptors
+
+import org.vitrivr.engine.core.database.Connection
+import org.vitrivr.engine.core.database.blackhole.BlackholeConnection
+import org.vitrivr.engine.core.database.descriptor.DescriptorProvider
+import org.vitrivr.engine.core.database.descriptor.DescriptorReader
+import org.vitrivr.engine.core.database.descriptor.DescriptorWriter
+import org.vitrivr.engine.core.model.descriptor.Descriptor
+import org.vitrivr.engine.core.model.metamodel.Schema
+
+/**
+ * A [DescriptorProvider] for the [BlackholeConnection].
+ *
+ * @author Ralph Gasser
+ * @version 1.0.0
+ */
+class BlackholeDescriptionProvider<T: Descriptor<*>>: DescriptorProvider<T> {
```

Comment:
Could be an object

Reply:
I don't think it could. Objects do not allow for generic type variables.